go/vt/vttablet/tabletmanager/tm_init.go

```diff
@@ -901,6 +901,10 @@ func (tm *TabletManager) initializeReplication(ctx context.Context, tabletType t
 	}
 
 	// Set primary and start replication.
+	if currentPrimary.Tablet.MysqlHostname == "" {
+		log.Warningf("primary tablet in the shard record doesn't have mysql hostname specified. probably because that tablet shutdown.")
```

Comment:
```suggestion
		log.Warningf("primary tablet in the shard record does not have mysql hostname specified, possibly because that tablet has been shut down.")
```